문자열내림차순으로배치하기.js

```diff
@@ -0,0 +1,5 @@
+function solution(s) {
+    let answer
+    answer = s.split('').sort().reverse().join('');
+    return answer;
```

Comment:
```suggestion
    return s.split('').sort().reverse().join('');
```
불필요하다고 생각되는 변수선언은 줄여주면 좋습니다.